### ISSUE TITLE
dependency check

### ISSUE THREAD

**Issue by kozliatko** [nextcloud/terms_of_service#31]
Hello
I installed fresh nextcloud 14.0 and enabled Terms of Services. After this I can not view content of my files, because I didn't have installed maxmind and ToS package didn't check dependency. 
In web browser I found only short error message
```
PDF.js v1.9.426 (build: 2558a58d)
Message: Unexpected server response (500) while retrieving PDF "https://my.server/remote.php/webdav/Nextcloud%20Manual.pdf".
```
but in nextcloud logs is

```
Sep 11 11:21:56 my Nextcloud[16866]: {PHP} Error: Class 'MaxMind\Db\Reader' not found at /var/www/nextcloud/apps/terms_of_service/lib/CountryDetector.php#46
```
Please can you add some dependency check for this plugin, because without access to log files on server is impossible to found source of this problem.

**Comment by nickvergessen** [comment]
How did you install it?

**Comment by kozliatko** [comment]
manually from https://download.nextcloud.com/server/releases/latest.tar.bz2 a few days ago (CentOS7 + remi php72)

**Comment by nickvergessen** [comment]
The terms of service app is not included there. You should either download it from https://apps.nextcloud.com/apps/terms_of_service or install it via the app management from inside your Nextcloud Administration.

I just checked it and that contains the lib.

**Comment by nextcloud101** [comment]
Hi, same problem here after installation with docker and snappy. 

Error: Class 'MaxMind\Db\Reader' not found at /var/www/nextcloud/apps/terms_of_service/lib/CountryDetector.php#46

**Comment by nextcloud101** [comment]
Installed the app via the app management like you said, but messages are still in the log file

**Comment by kozliatko** [comment]
@nickvergessen yes, ToS I installed via the app management. NextCloud I installed from latest source tar.
In app management there was no error during enabling.

**Comment by nickvergessen** [comment]
Indeed, now I managed to get the missing file aswell.
I just published 1.0.1 in the appstore, which should fix this issue:
https://apps.nextcloud.com/apps/terms_of_service

**Comment by kozliatko** [comment]
@nickvergessen now it works like a charm. Thank you very much.
